service/gcs/runtime/runc/runc.go

```diff
@@ -122,11 +122,12 @@ func (r *runcRuntime) CreateContainer(id string, bundlePath string, stdioSet *st
 // CreateContainer.
 func (c *container) Start() error {
 	logPath := c.r.getLogPath(c.id)
-	cmd := exec.Command("runc", "--log", logPath, "start", c.id)
+	cmd := exec.Command("runc", "--log", logPath, "--log-format", "json", "start", c.id)
```

Comment:
Would be nice to make a helper for this (`exec.Command("runc", "--log", logPath, "--log-format", "json", args...)`)

Reply:
Done. Let me know if my change is not what you had in mind.